packages/terra-core-docs/src/terra-dev-site/doc/pills/example/DisclosurePill.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useRef, useState } from 'react';
+import uniqueid from 'lodash.uniqueid';
+import Popup from 'terra-popup';
```

Comment:
It doesn't look like terra-core-docs has an explicit dependency on terra-popup, we should add one.